core/src/com/unciv/ui/utils/Popup.kt

```diff
@@ -27,7 +36,7 @@ open class Popup(val screen: CameraStageBaseScreen): Table(CameraStageBaseScreen
 
         this.isVisible = false
         touchable = Touchable.enabled // don't allow clicking behind
-        setFillParent(true)
+        this.setFillParent(true)
```

Comment:
? What does this add?

Reply:
Actually, good you bring this up as I should have. See `Note: "LeakingThis"` in #3971 - saves a `@Suppress("LeakingThis")` as it eliminates the warning, but I'm not 100% sure it's the correct approach.

Reply:
That's odd. I mean, it's not even *leaking* the this - it remains firmly within the boundaries of the class.
And if the point is "don't activate subclass functions in constructor" then why does adding the "this." help? I am even more confused :/ 
Not at you, but at the warning

Reply:
Yea - thing is as I understand you're in a constructor and neither this nor super is finished initializing, but your call might go to a class's method expecting something to be initialized that isn't. It's the fine detail that eludes me - who might wrongly expect what to be initialized? Problem is maybe just a missing final in libGdx...

Say, as discord-replacement: What would you think of the scraper script [here](https://github.com/SomeTroglodyte/Scratchpad/tree/master/Demos)? They _do_ look nice in-game: 
![image](https://user-images.githubusercontent.com/63000004/119711932-5ceb6280-be60-11eb-914c-20372b8fa0b8.png)

Back to code in a minute - there's an issue I'm having with minimap sizes I'm not sure is related to this branch, but I need to know for sure whether it is...